.circleci/config.yml

```diff
@@ -33,11 +33,15 @@ commands:
             }
 
             echo 'Checking changes'
-            changes=`git diff --name-only origin/master...$CIRCLE_BRANCH | { grep -Ev '^packages\/(?!storybook)|yarn.lock|bear.png|.editorconfig' || true; }`
-            echo 'Changes detected:'
-            echo $changes
+            globalChanges=`git diff --name-only origin/master...$CIRCLE_BRANCH | { grep -Ev '^packages/|yarn.lock|bear.png|.editorconfig' || true; }`
+            storybookChanges=`git diff --name-only origin/master...$CIRCLE_BRANCH | { grep '^packages/storybook' || true; }`
```

Comment:
Not the nicest solution, but we were having trouble with the previous regex, and we're also very close to deprecating CircleCI, so 🤠